syntax = "proto3";

package models;

message Role {
  int32 id = 1;
  string name = 2;
}
service RoleService {
  rpc CreateRole (CreateRoleRequest) returns (Role);
  rpc GetRoleById (GetRoleByIdRequest) returns (Role);
}

message CreateRoleRequest {
  string name = 1;
}

message GetRoleByIdRequest {
  int32 id = 1;
}
